webapp/src/js/enketo/medic-xpath-extensions.js

```diff
@@ -138,25 +145,54 @@ const addDate = function (date, years, months, days, hours, minutes) {
     [hours, 'hours'],
     [minutes, 'minutes'],
   ].filter(([value]) => value)
-    .map(([value, name]) => ([ +asString(value), name ]))
+    .map(([value, name]) => ([+asString(value), name]))
     .filter(([value]) => value)
     .forEach(([value, name]) => moment.add(value, name));
   return XPR.date(moment.toDate());
 };
 
+const dateDiff = function (startDateStr, endDateStr, key) {
+  key = getValue(key);
```

Comment:
I think this may have gotten missed! It was marked as resolved, but I don't see the change reflected.  (But let me know if I am missing something here and we actually need to be making this call!)

Reply:
Nice spot! Removed the if checks but never this part 🤦. Tested & changes pushed up.